src/editions/index.md

```diff
@@ -27,9 +27,10 @@ This serves different purposes for different people:
 When a new edition becomes available in the compiler, crates must explicitly opt
 in to it to take full advantage. This opt in enables editions to contain
 incompatible changes, like adding a new keyword that might conflict with
-identifiers in code, or turning warnings into errors. The Rust compiler can link
-crates of any editions together. Edition changes only affect the way the
-compiler initially parses the code. Therefore, if you're using Rust 2015, and
+identifiers in code, or turning warnings into errors. A Rust compiler can support
+multiple editions, and can link crates of any supported editions together.
```

Comment:
It would be good to say explicitly something akin to "A Rust compiler will support all editions that existed prior to the compiler's release" as this makes it sound like a Rust compiler can pick and chose.